app/controllers/application_controller.rb

```diff
@@ -60,6 +60,23 @@ class RecordExists < Exceptions::RecordExists
 
   around_action :run_with_transaction
 
+  # path_default_content_type configures default content-types from within
+  # application controllers.
+  #
+  # ==== Params:
+  # - path_match: Regular expression to match an incoming request path against.
+  # - content_type: String containing the content-type to use for a request
+  #                 to the matching path that does not contain a content-type
+  #                 header.
+  #
+  # ==== Examples:
+  #
+  # path_default_content_type /^\/policies/, 'application/x-yaml'
+  #
+  def self.path_default_content_type(path_match, content_type)
```

Comment:
Since this is mutating the variable and not just returning, let's use a verb name:

```
set_default_content_type_for_path
```